pkg/client/outofband/client.go

```diff
@@ -331,25 +331,36 @@ func didServiceBlockFunc(p Provider) func() (*did.Service, error) {
 
 		verKeyB58 := base58.Encode(verKey)
 
+		connections, err := routeSvc.GetConnections()
+		if err != nil {
+			return nil, fmt.Errorf("didServiceBlockFunc: create invitation - fetch router connections: %w", err)
+		}
+
+		if len(connections) == 0 {
+			return &did.Service{
+				ID:              uuid.New().String(),
+				Type:            "did-communication",
+				RecipientKeys:   []string{verKeyB58},
+				ServiceEndpoint: p.ServiceEndpoint(),
+			}, nil
+		}
+
 		// get the route configs
-		serviceEndpoint, routingKeys, err := mediator.GetRouterConfig(routeSvc, p.ServiceEndpoint())
+		serviceEndpoint, routingKeys, err := mediator.GetRouterConfig(routeSvc, connections[0], p.ServiceEndpoint())
```

Comment:
Same comment as DIDExchange client. ie, support for the consumers to specify the routers (could be >1).

Reply:
Sure, I will add it as well. It should not be a problem.

Reply:
Done! Added possibly to provide `RouterConnectionID`.